tests/src/JIT/HardwareIntrinsics/X86/Shared/SimdScalarUnOpConvTest.template

```diff
@@ -0,0 +1,341 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/******************************************************************************
+ * This file is auto-generated from a template file by the GenerateTests.csx  *
+ * script in tests\src\JIT\HardwareIntrinsics\X86\Shared. In order to make    *
+ * changes, please update the corresponding template and run according to the *
+ * directions listed in the file.                                             *
+ ******************************************************************************/
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace JIT.HardwareIntrinsics.X86
+{
+    public static partial class Program
+    {
+        private static void {Method}{Op1VectorType}{Op1BaseType}()
+        {
+            var test = new SimdScalarUnaryOpConvertTest__{Method}{Op1VectorType}{Op1BaseType}();
+
+            if (test.IsSupported && ({CanRunTests}))
```

Comment:
The other templates, rather than doing `{CanRunTests}` just had this handled in `IsSupported` as:
```
public bool IsSupported => {Isa}.IsSupported && (Environment.Is64BitProcess || ((typeof({RetBaseType}) != typeof(long)) && (typeof({RetBaseType}) != typeof(ulong))));
```

Reply:
This looks much better - I would prefer to correct it in #19670 to save on tests runs

Reply:
Will change it in this PR as correction from #19866 should go in first.